iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/ScalingParameters.java

```diff
@@ -80,4 +91,20 @@ public ScalingParameters setIterative(boolean iterative) {
         this.iterative = iterative;
         return this;
     }
+
+    public static ScalingParameters load() {
+        return load(PlatformConfig.defaultConfig());
+    }
+
+    public static ScalingParameters load(PlatformConfig platformConfig) {
+        Objects.requireNonNull(platformConfig);
+        ScalingParameters scalingParameters = new ScalingParameters();
+        platformConfig.getOptionalModuleConfig("scaling-default-parameters").ifPresent(config -> {
```

Comment:
Is there a rule for property names? (between scaling-default-parameters and scalingConvention)

Reply:
not really, I took inspiration from conventions in LoadFlowParameters... (camel for attributes and hyphens for the name)